packages/core/src/Asset/NftMetadata/fromMetadatum.ts

```diff
@@ -121,7 +121,11 @@ export const fromMetadatum = (
   const assetId = Cardano.AssetId.fromParts(asset.policyId, asset.name);
 
   if (version === '1.0' && !name) {
-    name = AssetName.toUTF8(asset.name);
+    try {
+      name = AssetName.toUTF8(asset.name);
+    } catch (error) {
+      logger.warn(error);
```

Comment:
We shouldn't continue in this case, as metadata is invalid

```suggestion
      logger.warn(error);
      return null;
```

Reply:
The `return null` is already in the next line